src/main/java/team/gif/robot/Robot.java

```diff
@@ -164,6 +163,6 @@ public void simulationPeriodic() {}
 
     //TODO: Change and check before each usage
     public static boolean isCompBot = false;
-    public static boolean isSwervePBot = false;
-    public static boolean isTankPBot = true;
+    public static boolean isSwervePBot = true;
```

Comment:
Are we going to stick with the swervePBot as main now?

Reply:
I'll update this PR to flip it back for the sake of consistency